js/mexc.js

```diff
@@ -590,33 +601,53 @@ module.exports = class mexc extends Exchange {
             const quantityScale = this.safeInteger (market, 'quantity_scale');
             const pricePrecision = 1 / Math.pow (10, priceScale);
             const quantityPrecision = 1 / Math.pow (10, quantityScale);
-            const precision = {
-                'price': pricePrecision,
-                'amount': quantityPrecision,
-            };
             const taker = this.safeNumber (market, 'taker_fee_rate');
             const maker = this.safeNumber (market, 'maker_fee_rate');
             const state = this.safeString (market, 'state');
-            const active = (state === 'ENABLED');
             const type = 'spot';
-            const swap = false;
-            const spot = true;
-            result.push (this.extend (this.fees['trading'], {
-                'info': market,
+            const fees = this.safeValue (this.fees, 'trading');
```

Comment:
We should not populate these, they're populated in setMarkets in the base class.